ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -1578,6 +1578,53 @@ literals:
 - valve_status:
   - OPEN
   - CLOSED
+
+- failed_panel_alarm:
+  - ACTIVE
+  - INACTIVE
+- low_smoke_alarm: # for aspirating type smoke detectors
+  - ACTIVE
+  - INACTIVE
+- medium_smoke_alarm: # for aspirating type smoke detectors
+  - ACTIVE
+  - INACTIVE
+- particle_concentration_alarm:
+  - ACTIVE
+  - INACTIVE
+- low_battery_alarm:
+  - ACTIVE
+  - INACTIVE
+- load_bypass_status:
+  - OFF
+  - ON
+- use_battery_status:
```

Comment:
can this just be `battery_status`, which already exists?

Reply:
sir, i thought about that actually, but the semantic would be wrong, i'm afraid. battery status could mean low, mid, or full (battery level), but what I need for site device is to mean: if the battery is "in use" or not by the device itself. This is for UPS. "battery_load_status" could be nearer but still not correct semantically because it talks about the one using the battery. If there is a better alternate for "use_battery_status" to mean "battery is in use" we'll change it.